grakn-test/test-integration/src/test/java/ai/grakn/test/engine/GraknEngineFailoverIT.java

```diff
@@ -73,10 +75,10 @@
     public static final DistributionContext engine1 = DistributionContext.startSingleQueueEngineProcess().port(7890).redisPort(redisURI.getPort());
 
     @ClassRule
-    public static final DistributionContext engine2 = DistributionContext.startSingleQueueEngineProcess().port(5678).redisPort(redisURI.getPort());
+    public static final DistributionContext engine2 = DistributionContext.startSingleQueueEngineProcess().port(5678).redisPort(5512);
 
     @ClassRule
-    public static final DistributionContext engine3 = DistributionContext.startSingleQueueEngineProcess().port(6789).redisPort(redisURI.getPort());
+    public static final DistributionContext engine3 = DistributionContext.startSingleQueueEngineProcess().port(6789).redisPort(5513);
```

Comment:
How does this work?

Reply:
Right so before hand we were stating a REAL redis which every engine shared. It did not matter which engine started it. 

Now we giving each engine their own FAKE redis. We give each one their own redis because without doing that you will have to control the startups of engines, which is a lot of bloat work for this test. I.e. you will have to make sure at least one starts fully before starting the others. This is just simpler.

Reply:
Ahh, I see. And `engine1` is different because that's the one we read the tasks from to check the test passed.